html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <div class="form-example">
+    <label for="email">Email: </label>
+    <div><input type="text" name="email" id="name" required></div>
+  </div>
+
+    <label for="select">Why is coding cool?: </label>
+    <div><select name="devices" id="select">
+      <option value="">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+
+    <div>
+      <p>Favorite Language (choose one)</p>
+      <input type="radio" name="favLang" id="html" value="html">
+      <label for="html">HTML</label>
+      <input type="radio" name="favLang" id="css" value="css">
+      <label for="css">CSS</label>
+      <input type="radio" name="favLang" id="javascript" value="javascript">
+      <label for="javascript">JavaScript</label>
+    </div>
+
+  <div>
+    <p>Would you like to sign up for more information?</p>
+<input type="checkbox" name="signup" value="yes" id="signup">
+<label for="signup">Sign me up!</label>
+</div>
+<div>
+  <label for="comments">Comments</label><br>
+  <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+</div>
+<input type="submit">
```

Comment:
Make sure to wrap your form elements in a `<form>` element and keep an eye on your indenting.